Cabal/Distribution/Types/PackageDescription.hs

```diff
@@ -398,6 +389,13 @@ enabledBuildInfos pkg enabled =
 -- * Utils
 -- ------------------------------------------------------------
 
+allBuildDepends :: PackageDescription -> [Dependency]
```

Comment:
Need docs, same with the other function below.

Reply:
FYI, in other PR I made `allBuildInfo` return all `BuildInfo`, also non-buildable. I don't know if they should be filtered there.

Reply:
`enabledBuildDepends` below does that.